git_machete/git_operations.py

```diff
@@ -335,15 +341,11 @@ def get_remotes(self) -> List[str]:
         return self.__remotes_cached
 
     def get_url_of_remote(self, remote: str) -> str:
-        return self._popen_git("config", "--get", f"remote.{remote}.url").strip()  # 'git remote get-url' method has only been added in git v2.5.1
+        return self.get_config_attr_or_none(f"remote.{remote}.url")  # 'git remote get-url' method has only been added in git v2.5.1
 
-    def get_organization_and_repository_name_of_remote(self, remote: str) -> Tuple[str, str]:
-        org = self.get_config_attr_or_none(f"machete.github.{remote}.organization")
-        repo = self.get_config_attr_or_none(f"machete.github.{remote}.repository")
-        if org and repo:
-            return org.strip(), repo.strip()
-        else:
-            raise MacheteException(f'Organization and repository name is unset for {remote}')
+    def get_organization_and_repository_name_of_remote(self, remote: str) -> OrganizationRepository:
+        return OrganizationRepository(organization=self.get_config_attr_or_none(f"machete.github.{remote}.organization"),
```

Comment:
Hmmm I'm not sure yet.... but shouldn't `machete.github.organization` and `machete.github.repository` be a "global" key in the sense that they just refer to a single GitHub repository used for all `git machete github` operations performed on the given local repository?

Reply:
Or in other words, making these keys remote-dependent (i.e. you can have `machete.github.origin.organization` and `machete.github.my-fork.organization`) gives more elasticity... but does the user really need to run `git machete github` against more than one GitHub repo in the given local repo?

Reply:
Note that e.g. `gh` CLI has this exact single-Github-repository logic: if your remotes correspond to multiple GitHub repositories, it lets you select the single GitHub remote to perform CLI operations agaist